File: FestivalJava/Networking/src/main/java/dto/BuyTicketsDTO.java
package dto;

import java.io.Serializable;

/**
 * Created by Sebi on 30-Mar-17.
 *
 * Used to transfer data for a transaction
 */
public class BuyTicketsDTO implements Serializable {
    private Integer idShow;
    private String clientName;
    private Integer numberOfTickets;
    private String username;

    public BuyTicketsDTO(Integer idShow, String clientName, Integer numberOfTickets, String username) {
        this.idShow = idShow;
        this.clientName = clientName;
        this.numberOfTickets = numberOfTickets;
        this.username = username;
    }

    public Integer getIdShow() {
        return idShow;
    }

    public void setIdShow(Integer idShow) {
        this.idShow = idShow;
    }

    public String getClientName() {
        return clientName;
    }

    public void setClientName(String clientName) {
        this.clientName = clientName;
    }

    public Integer getNumberOfTickets() {
        return numberOfTickets;
    }

    public void setNumberOfTickets(Integer numberOfTickets) {
        this.numberOfTickets = numberOfTickets;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }
}

File: FestivalC#/Festival/Networking/dto/BuyTicketsDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Networking.dto
{
    [Serializable]
    public class BuyTicketsDTO
    {
        private int idShow;
        private string clientName;
        private int numberOfTickets;
        private string username;

        public int IdShow
        {
            get
            {
                return idShow;
            }

            set
            {
                idShow = value;
            }
        }

        public string ClientName
        {
            get
            {
                return clientName;
            }

            set
            {
                clientName = value;
            }
        }

        public int NumberOfTickets
        {
            get
            {
                return numberOfTickets;
            }

            set
            {
                numberOfTickets = value;
            }
        }

        public string Username
        {
            get
            {
                return username;
            }

            set
            {
                username = value;
            }
        }

        public BuyTicketsDTO(int idShow, string clientName, int numberOfTickets, string username)
        {
            this.IdShow = idShow;
            this.ClientName = clientName;
            this.NumberOfTickets = numberOfTickets;
            this.username = username;
        }
    }
}
